test/h2_integration/certificate_rejected_test.exs

```diff
@@ -28,6 +28,10 @@ defmodule H2Integration.CerificateRejectedTest do
     config = Setup.create_h2_worker_config(Setup.server_host(), context[:port])
 
     {:error, actual_reason} = GenServer.start(Sparrow.H2Worker, config)
-    assert {:tls_alert, 'bad certificate'} == actual_reason
+    case actual_reason do
+      {:tls_alert, 'bad certificate'} -> :ok
```

Comment:
Can it be unified somewhere in code?

Reply:
Not sure if I understand but that is the only place in the code where `'bad certificate'` appears

Reply:
So now you 2 possible actual reasons, and my question is if you can make it one?